content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,55 @@ To learn how to restrict connection use, see [Access and Authentication][4].
 
 To connect to an arbitrary service, use the HTTP connection type. For authentication options and setup instructions, see [HTTP action][10].
 
+## Connection groups
+
+You can create groups of connections so that your workflows can authenticate into the correct account or accounts based on the given inputs. Connections can be grouped together only if they share the same integration (for example, you cannot group GCP and AWS connections within the same group). 
+
+You define the members of a connection group using a connection's _Identifier Tags_. For example, you can create a connection group consisting of AWS accounts that have the `env` tag.
+
+Each connection in the group must have a different set of tag values for the given Identifier Tags. For example:
+- `connectionA {env:staging}` and `connectionB {env:prod}` can be grouped together.
+- `connectionA {env:staging}` and `connectionC {env:staging}` can't be grouped, because the group would contain duplicate tag values.
+
+You can only add connections you have **Resolve Access** for, and you can only use connections within a group if you have Resolve Access for those connections. If you use a group with a connection to which you don't have Resolve Access _and_ when your workflow runs it points to that connection, the workflow fails with a `403 Forbidden` error. To fix this issue, you can:
```

Comment:
"You can only add connections you have **Resolve Access** for, and you can only use connections within a group if you have Resolve Access for those connections."

Is there a way to simplify this sentence?